test/Microsoft.Sbom.Api.Tests/Executors/SBOMComponentsWalkerTests.cs

```diff
@@ -23,16 +25,14 @@
 
 namespace Microsoft.Sbom.Api.Executors.Tests;
 
-using Microsoft.ComponentDetection.Orchestrator.Commands;
-using Microsoft.Sbom.Adapters.ComponentDetection;
-
 [TestClass]
 public class SBOMComponentsWalkerTests
 {
     private readonly Mock<ILogger> mockLogger = new Mock<ILogger>();
     private readonly Mock<IConfiguration> mockConfiguration = new Mock<IConfiguration>();
     private readonly Mock<ISbomConfigProvider> mockSbomConfigs = new Mock<ISbomConfigProvider>();
     private readonly Mock<IFileSystemUtils> mockFileSystem = new Mock<IFileSystemUtils>();
+    private readonly Mock<IPackageDetailsFactory> mockPackageDetailsFactory = new Mock<IPackageDetailsFactory>();
```

Comment:
Not for this PR since it's an existing pattern, but when possible it's best to avoid storing mocks on the class since this breaks parallel execution of tests.